python/requirements/ml/requirements_tune.txt

```diff
@@ -9,18 +9,21 @@ ConfigSpace==0.4.18
 dragonfly-opt==0.1.6
 flaml==1.1.1
 freezegun==1.1.0
-gluoncv==0.10.1.post0
+# Requires decord which is unavailable for arm64
+gluoncv==0.10.1.post0; platform_machine != "arm64"
```

Comment:
Generally looks good to me, anything we need to do to test this change?

Reply:
It does not affect our CI machines and docker images (or the parts that do are captured in the respective build processes), so once CI passes we should be fine to go. The current state for mac arm machines was that install didn't work at all, so this PR should be a strict improvement.